iso_config.yaml

```diff
@@ -0,0 +1,37 @@
+log_file: "/var/tmp/host-os-iso-creator.log"
```

Comment:
For now, I will keep it without the "default" node, I will start a discussion about whether it is worth to merge the config files or not.